editor.planx.uk/src/pages/PlatformAdminPanel.tsx

```diff
@@ -0,0 +1,167 @@
+
+const TeamData: React.FC<TeamData> = ({ data }) => {
+  return (
+    <StyledTeamAccordion primaryColour={data.primaryColour}>
+      <AccordionSummary
+        id={`${data.name}-header`}
+        aria-controls={`${data.name}-panel`}
+        expandIcon={<Caret />}
+        sx={{ pr: 1.5 }}
+      >
+        <Typography variant="h2">{data.name}</Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Grid
+          container
+          direction="row"
+          justifyContent="flex-start"
+          alignItems="flex-start"
+          spacing={3}
+        >
+          <Grid item xs={4}>
+            <SummaryListTable dense={true}>
+              <>
+                <Box component="dt">{"Slug"}</Box>
+                <Box component="dd">
+                  <code>
+                    {`/`}
+                    {data.slug}
+                  </code>
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Homepage"}</Box>
+                <Box component="dd">{data.homepage || <NotConfigured />}</Box>
+              </>
+              <>
+                <Box component="dt">{"Logo"}</Box>
+                <Box component="dd">
+                  {data.logo ? <Configured /> : <NotConfigured />}
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Favicon"}</Box>
+                <Box component="dd">
+                  {data.favicon ? <Configured /> : <NotConfigured />}
+                </Box>
+              </>
+            </SummaryListTable>
+          </Grid>
+          <Grid item xs={4}>
+            <SummaryListTable dense={true}>
+              <>
+                <Box component="dt">{"Planning constraints"}</Box>
+                <Box component="dd">
+                  {data.planningDataEnabled ? (
+                    <Configured />
+                  ) : (
+                    <NotConfigured />
+                  )}
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Article 4s"}</Box>
+                <Box component="dd">{"?"}</Box>
+              </>
+              <>
+                <Box component="dt">{"Reference code"}</Box>
+                <Box component="dd">
+                  {data.referenceCode || <NotConfigured />}
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Subdomain"}</Box>
+                <Box component="dd">{data.subdomain || <NotConfigured />}</Box>
+              </>
+            </SummaryListTable>
+          </Grid>
+          <Grid item xs={4}>
+            <SummaryListTable dense={true}>
+              <>
+                <Box component="dt">{"GOV.UK Notify"}</Box>
+                <Box component="dd">
+                  {data.govnotifyPersonalisation?.helpEmail || (
+                    <NotConfigured />
+                  )}
+                </Box>
```

Comment:
Rather than display the whole JSONB personalisation here, I think the email should be enough of a cue if a council is still on the default `example@council.gov` or has updated it to their own? 

Updating still a manual database step & Gov Notify admin panel step, but a good candidate to move all personalisation fields related to this to team settings soon!

Reply:
Yep for sure - this is a good solution 👍